Assignment4/Assignment4.js

```diff
@@ -0,0 +1,24 @@
+const fs = require('fs');
+
+function readAppendWrite (readFile, appendFile, writeFile) {
+  fs.readFile (readFile, function (err1, data1) {
+    if (err1) throw err1;
+    console.log(data1.toString())
+    console.log("----------------------------------------------");
+
+    fs.appendFile(appendFile, data1, function (err2) {
+      if (err2) throw err2;
+  
+      fs.readFile (appendFile, function (err3, data2) {
+        if (err3) throw err3;
+        
+        fs.writeFile(writeFile, data2, function (err3) {
+          if (err2) throw err2;
```

Comment:
Throwing err2 on raising err3 error. Any logic behind this, which we should know?
Problem Type: Typo Error